config/.htaccess

```diff
@@ -1,4 +1,4 @@
-<IfModule mod_authz_core.c>
```

Comment:
No, you write `<<` instead of `<`. There would be no diff here if files were the same :)